ebuild-writing/misc-files/patches/text.xml

```diff
@@ -293,6 +293,14 @@ files automatically. Alternatively, you can specify the <c>-E</c> option with
 <c>eapply</c> if you want to remove an empty file.
 </p>
 
+<p>
+You should also avoid adding commented lines in your patch to the actual
+changes to e.g. source code.  Patches show removed lines by prefixing them with
+a <c>-</c>, so no information is lost by simply deleting lines rather than
+commenting them out (which adds noise). This makes the patch shorter and
+more readable.
+</p>
```

Comment:
I don't understand. If the patch is intended for upstream and comments help understanding the added lines, why would we need the extra step of removing them again? That unnecessary workload for devs, and the benefit is only that the patch files in the gentoo repo are slightly smaller.

Reply:
Often, submitted patches with commented out lines add nothing and it would have been better to just remove said lines.

I think maybe my phrasing was unclear here? I am referring to patches which include commenting out lots of lines with no real use rather than simply removing them, not explanatory comments

Reply:
Yes, "adding commented lines" was confusing. Just say that removed lines should be removed, not commented out.

Reply:
And maybe don't refer to any comment char like `#` because it differs between languages.